client/src/component/CodeWarsCard.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
starting from React 17, the automatic JSX transformation no longer requires importing React in every file. You can simply write JSX code without importing React. So delete this line